Patch by Robert Scheck <robert@fedoraproject.org> for phpMyAdmin >= 3.4.7, which makes the php-gettext
library that is used configurable, which is helpful or even required for some vendors or downstreams. The
default is the internal library to keep backward compatibility. More details can be found at SourceForge:
https://sourceforge.net/tracker/?func=detail&aid=3433770&group_id=23067&atid=377410

--- phpMyAdmin-3.4.7/libraries/vendor_config.php			2011-09-14 13:48:13.000000000 +0200
+++ phpMyAdmin-3.4.7/libraries/vendor_config.php.gettext		2011-09-25 23:51:55.000000000 +0200
@@ -59,4 +59,10 @@
  * Default value for check for version upgrades.
  */
 define('VERSION_CHECK_DEFAULT', true);
+
+/**
+ * Path to gettext.inc file. Useful when you want php-gettext somewhere else,
+ * eg. /usr/share/php/gettext/gettext.inc.
+ */
+define('GETTEXT_INC', './libraries/php-gettext/gettext.inc');
 ?>
--- phpMyAdmin-3.4.7/libraries/select_lang.lib.php			2011-09-14 13:48:13.000000000 +0200
+++ phpMyAdmin-3.4.7/libraries/select_lang.lib.php.gettext		2011-09-25 23:54:41.000000000 +0200
@@ -373,7 +373,7 @@
 /**
  * Load gettext functions.
  */
-require './libraries/php-gettext/gettext.inc';
+require GETTEXT_INC;
 
 /**
  * @global string  interface language
